app/handlers/handlers_connectors.py

```diff
@@ -0,0 +1,276 @@
+import base64
+import time
+
+import kopf
+import kubernetes.client
+import pendulum
+
+from app.crds import TwingateConnectorCRD
+from app.handlers.base import success
+from app.settings import get_version
+
+ANNOTATION_LAST_VERSION_CHECK = "twingate.com/last-version-check"
+ANNOTATION_NEXT_VERSION_CHECK = "twingate.com/next-version-check"
+
+
+def get_connector_pod(
+    crd: TwingateConnectorCRD, tenant_url: str, image: str
+) -> kubernetes.client.V1Pod:
+    spec = crd.spec
+    name = crd.metadata.name
+
+    env_labels_version_policy = []
+    if spec.image_policy:
+        env_labels_version_policy = [
+            {
+                "name": "TWINGATE_LABEL_VERSION_POLICY_SCHEDULE",
+                "value": spec.image_policy.schedule,
+            },
+            {
+                "name": "TWINGATE_LABEL_VERSION_POLICY_SPEC",
+                "value": spec.image_policy.version,
+            },
+        ]
+
+    # fmt: off
+    pod_spec = {
+        "containers": [
+            {
+                "env": [
+                    {"name": "TWINGATE_LABEL_DEPLOYED_BY", "value": "operator"},
+                    {"name": "TWINGATE_LABEL_OPERATOR_VERSION", "value": get_version()},
+                    {"name": "TWINGATE_URL", "value": tenant_url},
+                    {"name": "TWINGATE_LOG_LEVEL", "value": "7"},
```

Comment:
We should pass this a parameter like `tenant_url`

Reply:
@liorr yes its on my list of things I want but on separate PR...  wanted to keep this Pr to minimum as its already big